__tests__/app.test.js

```diff
@@ -82,20 +82,20 @@ describe('GET article by id', () => {
         expect(typeof articles.topic).toBe('string')
         expect(typeof articles.created_at).toBe('string')
         expect(typeof articles.votes).toBe('number')
+        expect(typeof articles.comment_count).toBe('number')
```

Comment:
what number do you expect to be? You know exactly what article you expect back because you are requesting for a specific article